types/hls.js/index.d.ts

```diff
@@ -353,13 +353,29 @@ declare namespace Hls {
         error?: CustomLogger;
     }
 
-    interface AudioTrack {
```

Comment:
Why are you changing the interface name? This is a breaking change for anybody explicitly using this interface.

Reply:
See https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43335

Reply:
Ok. Looks good to me then.